.github/workflows/appsignals-e2e-test.yml

```diff
@@ -233,11 +204,14 @@ jobs:
           curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
           curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/client-call/
 
+      - name: Build Gradle
+        run: ./gradlew
+
```

Comment:
Do we need this? Do we have this in the canary? I assume this is to deal with gradle build failures, but not sure if this actually deals with it aside from removing it from the validation step so we can identify the failure root cause easier.